onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphTransformer.cpp

```diff
@@ -89,12 +78,7 @@
             // We need to predict whether the nodes will be assigned to the DML transformer by Lotus,
```

Comment:
>will be assigned to the DML transformer by Lotus [](http://example.com/codeflow?start=52&length=48)

Is the comment still true? Or should this be past tense now?

Reply:
Agreed.